applications/app/views/atomEmbed.scala.html

```diff
@@ -0,0 +1,29 @@
+@import conf.Static
+@import model.AtomPage
+@import conf.Configuration
+@import model.content.{ MediaAtom, MediaWrapper, StoryQuestionsAtom }
+@import views.html.fragments.atoms.{ media, storyquestions }
+
+@(page: AtomPage)(implicit request: RequestHeader, context: model.ApplicationContext)
+
+<!DOCTYPE html>
+<html lang="en-GB" class="gu-atom-embed-html">
+<head>
+    <meta charset="utf-8">
+    <title>@page.metadata.webTitle</title>
+    <base target="_top">
+    @fragments.atomPageHead(page)
+</head>
+<body>
+    @page.atom match {
+      case model: MediaAtom => {
+          @media(model, displayCaption = false, mediaWrapper = Some(MediaWrapper.EmbedPage))
+      }
+      case model: StoryQuestionsAtom => {
+          @storyquestions(model, isAmp = false)
+      }
+      case _ => {}
+    }
```

Comment:
I was more thinking about moving this logic in the `AtomPage` (ie: add a `body: HTML` field that would be overridden in `MediaAtomPage`, etc...). Then you can simply call `@page.body` here.
No need to match on the atom itself. The `page`, acting as a view model, has all information you need to render the template.